recipe/bld.bat

```diff
@@ -32,9 +32,9 @@ if errorlevel 1 exit 1
 ninja -v -C builddir -j %CPU_COUNT%
```

Comment:
Seems that `meson configure builddir` failed on win64:
`(%BUILD_PREFIX%) %SRC_DIR%>if errorlevel 1 exit 1`
Or the error is something else

Reply:
It was failing at the build step, but it seemed to have been a fluke.